spec/forms/state_file/md_tax_refund_form_spec.rb

```diff
@@ -61,8 +67,15 @@
           routing_number: "019456124",
           account_number: "12345",
           account_type: "checking",
-          bank_name: "Bank official",
-          account_holder_name: "Laney Knope"
+          account_holder_first_name: "Laney",
+          account_holder_middle_initial: "",
+          account_holder_last_name: "Knope",
+          account_holder_suffix: "",
+          joint_account_holder_first_name: "Janey",
+          joint_account_holder_middle_initial: "",
+          joint_account_holder_last_name: "Knope",
+          joint_account_holder_suffix: "",
+          has_joint_account_holder: 'yes',
```

Comment:
is this info required for the spec? i can't seem to find where it's used

Reply:
I think it's more that some info exists and then it's nulled out when the `payment_or_deposit_type` is `"mail"`-- which remidns me I should just fill every field with _something_